### PR TITLE
Implemented Skeleton Loader

### LINKED ISSUE #22: Skeleton Loader
Placeholder layout while content is loading.

### PR DESCRIPTION
**Pull request by rishabhrawat05**
---
name: Pull request
about: Maintain this format while making a PR
title: ""
labels: ""
assignees: ""
---

## Description

Hi! @Git21221 
I have implemented the Skeleton Loader React Component.

Fixes #22 

## Type of Change

Please delete options that are not relevant:

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ Yes] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (changes to documentation only)
- [ ] Chore (cleanup or minor improvement that does not change functionality)

## Checklist

- [Yes ] My code follows the style guidelines of this project.
- [ Yes] I have performed a self-review of my code.
- [ Yes] I have commented my code, particularly in hard-to-understand areas.
- [No ] I have made corresponding changes to the documentation.
- [ Yes] My changes generate no new warnings or errors.
- [ Yes] I have added tests that prove my fix is effective or that my feature works.
- [ Yes] New and existing unit tests pass locally with my changes.

## Screenshots (if applicable)
Screenshot (4)

## Additional Information

Please add necessary label to this pull request

### CODE REVIEW

File: src/components/Skeleton/Skeleton.tsx
Comment (Git21221) [inline src/components/Skeleton/Skeleton.tsx:1]: width,
  height,
  animationDuration,
  variant = "rounded",
  size = "medium",
  animation,
  borderRadius = 25,
  gradientColors = ["#f0f0f0", "#e0e0e0", "#f0f0f0"], 
  backgroundColor = "lightgrey",
  
  you have used all these but in test/App.jsx you have checked only two (variant, animation)

Comment (Git21221) [reply src/components/Skeleton/Skeleton.tsx:1]: check if all are working and properly overriding default ones. and send screenshot of those too

Comment (rishabhrawat05) [reply src/components/Skeleton/Skeleton.tsx:1]: I have checked all parameters all are working fine. I will share you the screenshot later

Comment (Git21221) [reply src/components/Skeleton/Skeleton.tsx:1]: yeah Need to check before merging

**Comment by rishabhrawat05** [comment]
Hey! @Git21221 
Here is the screenshot and updated code
Screenshot (5)

**Comment by Git21221** [comment]
@rishabhrawat05 if I give gradientColors = ["", "", ""] then what will happen?

**Comment by rishabhrawat05** [comment]
> @rishabhrawat05 if I give gradientColors = ["", "", ""] then what will happen?

nothing the color will set to default color

image

here the last is with gradientColors=["","","",]

**Comment by Git21221** [comment]
@rishabhrawat05 ok fine

**Review by Git21221 — APPROVED** [review]

**Pull request merged**